UltimateTeam.Toolkit/Models/IListResponse.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+
+namespace UltimateTeam.Toolkit.Models
+{
+    public interface IListResponse
```

Comment:
What's the purpose of this interface? `AuctionResponse` and `WatchlistResponse` could share a base class or similar, but can't see why we need this.